os/include/tinyara/pm/pm.h

```diff
@@ -313,6 +307,12 @@ enum pm_timer_type_e {
 	/* Scope for future expansion, up to 8 timer types can be supported */
 };
 
+/* This structure is used to send data to pm driver from app side */
+struct pm_header_s {
```

Comment:
why it is header??

how about argument or other word?

Reply:
updated